modules/apps/batch-engine/batch-engine-test/src/testFunctional/tests/ExportOfJsontIncludeConfigurationObject.testcase

```diff
@@ -45,7 +45,8 @@ definition {
 		task ("Then the jsont format file contains configuration node with className 'com.liferay.object.rest.dto.v1_0.ObjectEntry' parameter") {
 			BatchEngine.assertExportedJsontContainsCorrectConfigurationObject(
 				expectedValue = "com.liferay.object.rest.dto.v1_0.ObjectEntry",
-				jsonObject = "className");
+				fileName = "export.batch-engine-data.json",
```

Comment:
Actually, as we talked yesterday, we don't need to pass export.batch-engine-data.json and this will be defined in BatchEngine.assertExportedJsontContainsCorrectObject.